functions/function/http-request-function/pom.xml

```diff
@@ -19,6 +19,11 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>org.springframework.cloud.fn</groupId>
+            <artifactId>config-common</artifactId>
```

Comment:
What does this `common` bring for us into this function?

Reply:
Used to bind Expression properties